servicetalk-data-jackson-jersey/src/test/java/io/servicetalk/data/jackson/jersey/AbstractStreamingJsonResourcesTest.java

```diff
@@ -47,6 +50,14 @@ public AbstractStreamingJsonResourcesTest(final RouterApi api) {
                     PublisherJsonResources.class
             ));
         }
+
+        @Override
+        public Map<String, Object> getProperties() {
+            Map<String, Object> properties = super.getProperties();
+            Map<String, Object> newProperties = new HashMap<>(properties);
+            newProperties.put(InternalProperties.JSON_FEATURE, "ServiceTalkJacksonSerializerFeature");
+            return newProperties;
```

Comment:
- Does `newProperties` have to be mutable? If not, consider using `singletonMap`
- Can you use the `ST_JSON_FEATURE` constant instead of `"ServiceTalkJacksonSerializerFeature"`?
- Please use static imports for readability (`JSON_FEATURE`)